core/src/test/scala/com/pingcap/tispark/datasource/RegionSplitSuite.scala

```diff
@@ -0,0 +1,30 @@
+package com.pingcap.tispark.datasource
+
+import com.pingcap.tikv.TiBatchWriteUtils
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{IntegerType, StructField, StructType}
+
+class RegionSplitSuite extends BaseDataSourceTest("region_pre_split_test") {
+  private val row1 = Row(1)
+  private val schema = StructType(
+    List(
+      StructField("a", IntegerType)
+    )
+  )
+
+  test("region pre split test") {
+    dropTable()
+    jdbcUpdate(
+      s"CREATE TABLE  $dbtable ( `a` int(11) DEFAULT NULL) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin"
+    )
+
+    val dbName = "tidb_tispark_test"
+    val tableName = "region_pre_split_test"
+    val options = Some(Map("enableRegionPreSplit" -> "true"))
+    tidbWrite(List(row1), schema, options)
+    val tiTableInfo =
+      ti.tiSession.getCatalog.getTable(dbName, tableName)
```

Comment:
change to `ti.tiSession.getCatalog.getTable(database, table)` and delete line 21-22

Reply:
Nope. It does not work. I already tried. NPE will be thrown due to `tidb_` prefix.

Reply:
my mistake, change to 
```
ti.tiSession.getCatalog.getTable(this.dbPrefix + database, table)
```